### ISSUE TITLE
Make Harvestcraft a required mod

### ISSUE THREAD

**Issue by SvenKayser** [reteo/Survival-Industry#130]
Considering how many things tie into Harvestcraft (especially considering OreDicts), the fact that Hunger Overhaul is a required mod as well, and considering Harvestcraft's lightweight nature, Harvestcraft should be an integral part of SI.

**Comment by reteo** [comment]
Some people prefer different food systems, such as the Plants MegaPack.  I'd rather not take the option away from them to disable PHC if they want to use something else.

**Comment by SvenKayser** [comment]
Fair enough. But in this case we will need to implement a gap-closer for some ore dictionary entries. I'll see about that.

**Comment by reteo** [comment]
If you want, you can attack the "Progressive Food" script, and see if you can use it to make something functional with a number of recipes to meet dietary requirements.

**Comment by SvenKayser** [comment]
Will do when I find the time. I really like the concept.
